evennia/commands/default/building.py

```diff
@@ -1558,7 +1562,7 @@ class CmdTypeclass(COMMAND_DEFAULT_CLASS):
     set or change an object's typeclass
 
     Usage:
-      @typclass[/switch] <object> [= <typeclass.path>]
+      @typeclass[/switch] <object> [= typeclass_path]
```

Comment:
I think the dot was to imply that the typeclass path can have a tree-like path with sections separated by dots and be rather long, like `evennia.contrib.tutorial_world.rooms.WeatherRoom` for example.  I can't think of a better way to denote that, though. Your changes here are rather helpful.